src/cart/src/cart/crt_context.c

```diff
@@ -463,10 +465,10 @@ crt_context_destroy(crt_context_t crt_ctx, int force)
 			"d_hash_table_traverse failed rc: %d.\n",
 			ctx->cc_idx, force, rc);
 		/* Flush SWIM RPC already sent */
-		rc = crt_context_flush(crt_ctx, CRT_SWIM_RPC_TIMEOUT);
+		rc = crt_context_flush(crt_ctx, crt_swim_rpc_timeout);
 		if (rc)
 			/* give a chance to other threads to complete */
-			sleep(CRT_SWIM_RPC_TIMEOUT);
+			sleep(crt_swim_rpc_timeout);
```

Comment:
We cannot monitor RPC in flight instead of a random sleep?

Reply:
We can, but we need to give a chance to complete anyway. The idea here is
1. we stop emitting new RPC
2. sleeping a timeout to complete already sent
3. cleanly stop

Reply:
Actually the error inside crt_context_flush() means we have a RPC in the flight. And we sleep only in this case.

Reply:
ok, then we should have a loop with a sleep(1ms) that verify regularly when the RPC has completed instead of a global long timeout.